en/ch06.md

```diff
@@ -4,24 +4,41 @@ The `Settings` tab provides tools for customizing configuration options.
 ### Wallet
 ![settings-wallet](media/black_settings-wallet.png)
 
+(1) **Close this wallet:** Click here to safely close the current wallet  
+(2) **Create a view-only wallet:** Click here to generate a view-only (or audit) wallet, which is capable to see existing outgoing transactions and all incoming transactions. See this [user-guide](https://getmonero.org/resources/user-guides/view_only.html) for more informations.  
```

Comment:
>which is capable to see existing outgoing transactions and all incoming transactions

I'd remove the existing outgoing transactions part, as that only applies after key images have been imported. Thus, change to:

>Click here to generate a view-only (or audit) wallet, which is capable of seeing all incoming transactions to the wallet. 

Perhaps we could add something later about importing key images. 

"informations" <= Change to information.

Reply:
> I'd remove the existing outgoing transactions part, as that only applies after key images have been imported.

Aren't the key images from previous transactions already included when you create a view-only wallet from a regular wallet?

Reply:
> "informations" <= Change to information.

Ok.

Reply:
That kind of complicates matters, as it would see outgoing transactions if the transaction used outputs that were received prior to the creation date, but not if the outputs were received after the creation date. I'll leave it up to you what to choose :-P

Reply:
I'll pass it to @erciccione then. I like being precise, but i get your point.

Reply:
Ok to keep it for me and then add the specification dEBRUYNE suggested. Maybe at the bottom of the article, to not make that section too heavy.

Reply:
You mean about "importing key images"? Then maybe in the chapter 8, after the notes about remote & bootstrap nodes? We could link here from chapter 3 (send Monero > Export key images) and from this chapter.

Reply:
Yes, we could also add some more info about key images, or link to the moneropedia.

Reply:
Indeed, add a bunch of info about what key images are, why they are needed for an audit (or view only) wallet, links to moneropedia, and an `import key images` screenshot.  
I'll will just wait for this #119 #122 and this one to be merged to work on the chapter 8 update, and add links to them in the same PR.